### ISSUE TITLE
[$20]Accounts App Dev Environment: Field highligth Indicator is misplaced

### ISSUE THREAD

**Issue by nithyaasworld** [topcoder-platform/community-app#3299]
Screen1:
1. In the Password Reset screen, enter invalid email address
2. Check the Field highlighter

Screen2:
1. In Create new password screen, check Field highlighter 

Actual:
They are placed down the field.

Expected:
It should be inside the field like in the production.

![image](https://user-images.githubusercontent.com/5114190/64453118-7933f280-d105-11e9-88c1-69574abfb84f.png)

![image](https://user-images.githubusercontent.com/5114190/64453131-7f29d380-d105-11e9-9820-28cc54823de0.png)

**Comment by veshu** [comment]
@nithyaasworld Confirmed there is issue with forgot password.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30101701 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30101701 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by nithyaasworld** [comment]
@veshu Thanks. Now 'Create New Password' doesn't have any field highlighter. But this is just like in the production.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30101701<br/><br/>```This is an automated message for crazyk via Topcoder X```
